cg/meta/workflow/rnafusion.py

```diff
@@ -163,57 +164,28 @@ def get_references_path(self, genomes_base: Optional[Path] = None) -> Path:
             return genomes_base
         return Path(self.references)
 
-    def get_verified_arguments(
-        self,
-        case_id: str,
-        input: Optional[str],
-        outdir: Optional[str],
-        genomes_base: Optional[str],
-        trim: bool,
-        fusioninspector_filter: bool,
-        all: bool,
-        pizzly: bool,
-        squid: bool,
-        starfusion: bool,
-        fusioncatcher: bool,
-        arriba: bool,
-        cli: bool = False,
-    ) -> Dict[str, str]:
-        """Transforms click argument related to rnafusion that were left empty into
-        defaults constructed with case_id paths or from config."""
-
-        prefix: str = "--" if cli else ""
-
+    def get_default_parameters(self, case_id: str) -> Dict:
+        """Returns a dictionary with default RnaFusion parameters."""
         return {
-            f"{prefix}input": NextflowAnalysisAPI.get_input_path(
-                case_id=case_id, root_dir=self.root_dir, input=input
+            "input": NextflowAnalysisAPI.get_input_path(
```

Comment:
@henrikstranneheim Do we want to move this to modules also for automatic type checking?

Reply:
@ChrOertlin is it ok if I resolve this for now and we address this later if it needs to be addressed?

Reply:
For a future PR; I was just wondering since I am unfamiliar with this part of the code and we might want to track these things.